src/pages/Liquidated.tsx

```diff
@@ -29,50 +22,31 @@ export function Liquidated() {
   };
 
   //  Queries for graph
-  const { data: tokenNamesData } = useSWR<AxiosResponse, AxiosError>(LIQUIDATION_GRAPH_TOKENS_QUERY, subQueryFetcher);
-  const tokenNamesResponse: TokenNamesResponse = tokenNamesData?.data.data;
-  const tokenNames = tokenNamesResponse?.vaultManagerMetrics.nodes.map((node) => node.liquidatingCollateralBrand) || [];
-
-  const { data: dailyMetricsData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
-    LIQUIDATION_DAILY_METRICS_QUERY(tokenNames),
-    subQueryFetcher,
+  const { data: vaultStateDailyData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
+    VAULT_STATE_DAILIES_QUERY,
+    (query: string) => subQueryFetcher(query, SUBQUERY_STAGING_URL),
   );
-  const dailyMetricsResponse: LiquidationMetricsDailyResponse = dailyMetricsData?.data.data;
+  const vaultStateDailyResponse: VaultStateDailyResponse = vaultStateDailyData?.data.data;
   const graphDataMap: { [key: number]: GraphData } = {};
-  Object.values(dailyMetricsResponse || {}).forEach((tokenDataList) => {
-    tokenDataList?.nodes.forEach((dailyTokenMetrics) => {
-      graphDataMap[dailyTokenMetrics.dateKey] = {
-        ...graphDataMap[dailyTokenMetrics.dateKey],
-        x: dailyTokenMetrics.blockTimeLast.split('T')[0],
-        key: dailyTokenMetrics.dateKey,
-        active: {
-          ...graphDataMap[dailyTokenMetrics.dateKey]?.active,
-          [dailyTokenMetrics.liquidatingCollateralBrand]: dailyTokenMetrics.numActiveVaultsLast,
-        },
-        liquidated: {
-          ...graphDataMap[dailyTokenMetrics.dateKey]?.liquidated,
-          [dailyTokenMetrics.liquidatingCollateralBrand]: dailyTokenMetrics.numLiquidationsCompletedLast,
-        },
+  vaultStateDailyResponse?.vaultStatesDailies.nodes?.forEach((vaultState) => {
+      graphDataMap[Number(vaultState.id)] = {
+        x: vaultState.blockTimeLast.split('T')[0],
+        key: Number(vaultState.id),
```

Comment:
Perhaps optional chaining operator with `vaultState.blockTimeLast.split('T')[0],` for the just in case

Reply:
we should only be putting optional chaining where we expect an object could be null. the object shouldnt be null here

Reply:
In the past, we've encountered cases where data from the indexer is missing or contains empty objects. I suggest adding the optional chaining operator wherever there's a just-in-case chance of data being absent.